openslides/agenda/models.py

```diff
@@ -163,7 +163,7 @@ def get_title(self):
         Return the title of this item.
         """
         if not self.content_object:
-            agenda_title = self.title
+            agenda_title = self.title or ""
```

Comment:
Why don't we send just 'None' so that the client knows that there isn't any title but the title of the related item?